run-sandwich-run/index.html

```diff
@@ -0,0 +1,81 @@
+<!doctype html>
+
+<html>
+
+<head>
+  
+  <title>Run, Sandwich, Run!</title>
+  
+  <meta name="title" content="Run, Sandwich, Run!">
+  <meta name="author" content="Chad Sansing">
+  <meta charset="utf-8">
+  
+  <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.0.0/jquery.min.js"></script>
+  
+  <script src="scripts/breads.js" type="text/javascript"></script>
+  <script src="scripts/cheeses.js" type="text/javascript"></script>
+  <script src="scripts/dressings.js" type="text/javascript"></script>
+  <script src="scripts/proteins.js" type="text/javascript"></script>
+  <script src="scripts/veggies.js" type="text/javascript"></script>
+  <script src="scripts/score.js" type="text/javascript"></script>
+  <script src="scripts/fates.js" type="text/javascript"></script>
+  <script src="scripts/adj.js" type="text/javascript"></script>
+  <script src="scripts/distance.js" type="text/javascript"></script>
```

Comment:
After you generate sandwiches, they race for their lives. These scripts describe their fates.